source/reference/compatibility.txt

```diff
@@ -97,8 +98,11 @@ To specify a read preference, you can use either:
 
 - The :method:`Mongo.setReadPref()` method.
 
-The legacy ``mongo`` shell required running ``rs.secondaryOk()`` to
-enable secondary reads.
+- When using ``mongosh`` to connect directly to a :term:`secondary` 
+  replica set member, if your read preference is set to 
+  :readmode:`primaryPreferred`,:readmode:`secondary` or 
```

Comment:
Hey @ianf-mongodb , this has some rendering issues in staging. Can you check?

https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-19231-add-clarity-to-read-operations-on-secondary-nodes/reference/compatibility/#read-operations-on-a-secondary-node

Reply:
I had a missed space :readmode:`primaryPreferred`,:readmode:`secondary` which was causing the render issue. Fixed.